docs/quickstarts/express.mdx

```diff
@@ -0,0 +1,173 @@
+
+<Steps>
+  ### Install `@clerk/express`
+
+  Clerk's [Express SDK](/docs/references/express/overview) has prebuilt components, React hooks, and helpers to make user authentication easier.
+
+  To get started using Clerk with Express, add the SDK to your project:
+
+  <CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install @clerk/express
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add @clerk/express
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add @clerk/express
+    ```
+  </CodeBlockTabs>
+
+  ### Set your Clerk API keys
+
+  <SignedIn>
+    Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+  </SignedIn>
+
+  <SignedOut>
+    1. Navigate to the Clerk Dashboard.
+    1. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+    1. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+    1. Paste your keys into your `.env.local` file.
+
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```sh {{ filename: '.env.local' }}
+  CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+  This guide uses `dotenv` to load the environment variables. Install the package by running the following command:
+
+  <CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install dotenv
+    npm install -D @types/dotenv
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add dotenv
+    yarn add -D @types/dotenv
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add dotenv
+    pnpm add -D @types/dotenv
+    ```
+  </CodeBlockTabs>
+
+  ### Add middleware to your application
+
+  The [`clerkMiddleware()`](/docs/references/express/overview#clerk-middleware) helper integrates Clerk authentication into your Express application. It can be registered globally or for specific routes. The following example registers it globally.
+
+  ```ts {{ filename: 'index.ts', mark: [3, 8] }}
+  import 'dotenv/config'
+  import express from 'express'
+  import { clerkMiddleware } from '@clerk/express'
+
+  const app = express()
+  const PORT = 3000
+
+  app.use(clerkMiddleware())
+
+  app.listen(PORT, () => {
+    console.log(`Example app listening at http://localhost:${PORT}`)
+  })
+  ```
+
+  ### Accessing auth state using `getAuth()`
+
+  The [`getAuth()`](/docs/references/express/overview#get-auth) helper retrieves authentication state from the request object.
+
+  [`clerkClient`](/docs/references/express/overview#clerk-client) is an instance of Clerk's JavaScript Backend SDK, which provides access to Clerk's Backend API resources and low-level authentication utilities for JavaScript environments.
```

Comment:
because `clerkClient` is used in the code example, we should probably explain to the user what it is and what its doing. if that means the title of the section needs to change, thats true - but i wanted this copy added for the sake of the code example.